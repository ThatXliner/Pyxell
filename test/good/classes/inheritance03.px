
class A def
    func f(): String def
        return "A.f"

    func g(): String def
        return "A.g"

class B(A) def
    func f(): String def
        return "B.f"

    func h(): String def
        return "B.h"

a = A()
print a.f(), a.g()

b = B()
print b.f(), b.g(), b.h()
a = b
print a.f(), a.g()
