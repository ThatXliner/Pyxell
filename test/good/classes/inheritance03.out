A.f A.g
B.f A.g B.h
B.f A.g
